### PR TITLE
refactor(seeder): sidebar

### PR DESCRIPTION
**Pull request by moonlitgrace**

### CODE REVIEW

File: seeder/src/renderer/main.css [hunk 1]
```diff
@@ -27,4 +27,5 @@
 * {
 	outline: none !important;
 	box-sizing: border-box;
+  user-select: none;
 }
```
Comment (baseplate-admin) [inline seeder/src/renderer/main.css:31]: we are using tailwind right?

**Review by baseplate-admin — CHANGES_REQUESTED** [review]

File: seeder/src/renderer/main.css (hunk 1 above)
Comment (moonlitgrace) [reply seeder/src/renderer/main.css:31]: we are? lemme see... :confused:

Comment (baseplate-admin) [reply seeder/src/renderer/main.css:31]: Kindly remove all these from here and use tailwind class

Comment (moonlitgrace) [reply seeder/src/renderer/main.css]: Resolved on https://github.com/coreproject-moe/CoreProject/pull/3929/commits/d47db09259a7c83064d00cb08b7337bc87afedf2

File: seeder/src/renderer/main.css [hunk 2]
```diff
@@ -25,7 +25,5 @@
...
 
 * {
-	outline: none !important;
-	box-sizing: border-box;
-  user-select: none;
+  @apply !outline-none select-none;
```
Comment (baseplate-admin) [inline seeder/src/renderer/main.css:28]: I am still not a fan of this method. Can you do it in components selectively?

**Review by baseplate-admin — CHANGES_REQUESTED** [review]

File: seeder/src/renderer/main.css (hunk 2 above)
Comment (moonlitgrace) [reply seeder/src/renderer/main.css:28]: It is a pain you know right?
what feels off?

Comment (baseplate-admin) [reply seeder/src/renderer/main.css:28]: >  select-none;

Comment (moonlitgrace) [reply seeder/src/renderer/main.css]: https://github.com/coreproject-moe/CoreProject/pull/3929/commits/d4c4c9fc3bee6bf59108af9360572e15d72b7eb5: `select-none` can be added to body el, and for elements you want to select: `select-text/auto` can be added

File: seeder/src/renderer/App.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+				component: lazy(() => import("@routes/anime/themes"))
+			}
+		]
+	}
+];
+
+const App: Component = () => {
+	onMount(() => {
+		defineCustomElements();
+	});
```
Comment (baseplate-admin) [inline seeder/src/renderer/App.tsx:49]: I dont get it?

**Review by baseplate-admin — CHANGES_REQUESTED** [review]

File: seeder/src/renderer/App.tsx (hunk 3 above)
Comment (baseplate-admin) [reply seeder/src/renderer/App.tsx:49]:
```tsx
import "./main.css";

import { render } from "solid-js/web";
import { RouteDefinition, Router } from "@solidjs/router";
import { lazy } from "solid-js";
import { defineCustomElements } from "@coreproject-moe/icons/components";

const routes: RouteDefinition[] = [
	{
		path: "/",
		component: lazy(() => import("@layouts/RootLayout")),
		children: [
			{
				path: "",
				component: lazy(() => import("@routes/home"))
			},
			{
				path: "/character-urls",
				component: lazy(() => import("@routes/character"))
			},
			{
				path: "/demographics",
				component: lazy(() => import("@routes/demographics"))
			},
			{
				path: "/staff-urls",
				component: lazy(() => import("@routes/staff"))
			},
			{
				path: "/anime-explicit-genres",
				component: lazy(() => import("@routes/anime/explicit-genres"))
			},
			{
				path: "/anime-genres",
				component: lazy(() => import("@routes/anime/genres"))
			},
			{
				path: "/anime-urls",
				component: lazy(() => import("@routes/anime"))
			},
			{
				path: "/anime-themes",
				component: lazy(() => import("@routes/anime/themes"))
			}
		]
	}
];
defineCustomElements();
render(() => <Router>{routes}</Router>, document.getElementById("root") as HTMLElement);

```

**Pull request merged**